src/logger/transports/bitdrift.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  debug as bdDebug,
+  error as bdError,
+  info as bdInfo,
+  warn as bdWarn,
+} from '#/lib/bitdrift'
+import {LogLevel, Transport} from '#/logger/types'
+import {prepareMetadata} from '#/logger/util'
+
+const logFunctions = {
+  [LogLevel.Debug]: bdDebug,
+  [LogLevel.Info]: bdInfo,
+  [LogLevel.Log]: bdInfo,
+  [LogLevel.Warn]: bdWarn,
+  [LogLevel.Error]: bdError,
+} as const
+
+export const bitdriftTransport: Transport = (
+  level,
+  context,
+  message,
+  metadata,
+) => {
+  const log = logFunctions[level]
+  log(`(${context}) ${message}`, prepareMetadata(metadata))
```

Comment:
(hi from bitdrift 👋 ) not sure what context is but it could also be a good idea to also pass that as a field (aka metadata) so bitdrift can then group by/filter by that value /passerby comment

Reply:
Thanks! Yeah was considering that also. Initially I wanted the logs to match our dev-only console transport, but that's not really a big deal and it will def work better with Bitdrift. Sentry I'm less sure on, but not too worried about Sentry right now.